src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Don't need to use inheritance. There is no requirement to have `String getRandomColor()` in this class. Let's use composition instead